amity_app/classes/amity.py

```diff
@@ -304,58 +279,39 @@ def create_room(self, room_type, room_names):
                 Amity.living_spaces[name] = LivingSpace(name)
                 successfully_created_rooms.append(name)
 
-        if len(successfully_created_rooms):
+        if successfully_created_rooms:
             success_msg = "\n Successfully created " + room_type.lower() + "s: " + \
                           str(len(successfully_created_rooms))\
                           + " ,\n " + " ".join(successfully_created_rooms) + "\n"
 
-        if len(format_error_room_names):
+        if format_error_room_names:
             error_msg = "\n Room names rejected due to format errors: " + \
                         str(len(format_error_room_names)) + " ,\n "\
                         + " ".join(format_error_room_names) + "\n"
 
-        if len(already_existing_rooms):
+        if already_existing_rooms:
             failure_msg = "\n Room names rejected because they already exist: " + \
                           str(len(already_existing_rooms)) +\
                           " ,\n " + " ".join(already_existing_rooms) + "\n"
 
         return success_msg + error_msg + failure_msg
 
     # confirms whether user has entered a valid identifier
-    def confirm_person_identifier(self, person_identifier):
+    def confirm_person_identifier_exists(self, person_identifier):
```

Comment:
this function should be handled inside the function to get a person using his identifier. that function should first check the identifier exists. Consider removing this function.